### PR TITLE
Makes strong booze more dangerous

### PR DESCRIPTION
**Pull request by Mickyan**
:cl:
balance: Strong alcoholic beverages in excessive quantity are now far more dangerous. Drink responsibly.
/:cl:

I think something changed at some point a while ago (liver updates maybe) and it's almost impossible to die from alcohol poisoning without actually trying. 
I did some research: on average, drinking an entire bottle of whiskey all at once has a good chance of killing you, so I set to replicate that through testing. 

Drinking any more than that is going to start damaging you and possibly slowly kill you

I had to slightly alter some of the effects of being drunk for this to work

### CODE REVIEW

**Comment by adamkad1** [comment]
what about alcohol resistant? or whatever its called

**Comment by as334** [comment]
This is a blatant detective nerf and I won't stand for it.

**Comment by Farquaar** [comment]
Absinthe is like, 40-70% alcohol, but people chug multiple bottles down without any dilution. Hopefully this change will make stronger alcohol feel more distinct from your average space beer.

**Comment by Tlaltecuhtli** [comment]
he would die but it takes time for the liver to start falling as its metabolism based (if you take 50u at the time you need to process at least 20u to start being fuck

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#define ALCOHOL_THRESHOLD_MODIFIER 0.05 //Greater numbers mean that less alcohol has greater intoxication potential
+#define ALCOHOL_THRESHOLD_MODIFIER 0.5 //Greater numbers mean that less alcohol has greater intoxication potential
 #define ALCOHOL_RATE 0.005 //The rate at which alcohol affects you
-#define ALCOHOL_EXPONENT 1.6 //The exponent applied to boozepwr to make higher volume alcohol atleast a little bit damaging.
+#define ALCOHOL_EXPONENT 1.725 //The exponent applied to boozepwr to make higher volume alcohol atleast a little bit damaging to the liver
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Could you change the "atleast" to "at least" while you're editing this line?

**Comment by ATH1909** [comment]
Isn't this a MASSIVE bartender nerf?

How're you supposed to get people to hang out and drink at your bar if they can only stomach one drink before passing out/almost dying?

Doesn't this cause most of the "utility" drinks to be near-useless now, because they'll all kill you in relatively small quantities?

**Comment by Mickyan** [comment]
A hundred units of one of the strongest drinks in the game is not a small amount and the lower metabolization rate makes up for it

**Comment by Mickyan** [comment]
I changed to a better approach that is a little more forgiving, this shouldn't be an issue for anybody that isn't looking to chug multiple bottles of absinthe on the spot

**Comment by necromanceranne** [comment]
I wonder how lethal Bacchus might have become.

**Comment by dr-bee** [comment]
> 
> 
> I wonder how lethal Bacchus might have become.

This would probably make a shot of it lethal instead of just enough to get you shitfaced.

**Comment by ATH1909** [comment]
> > I wonder how lethal Bacchus might have become.
> 
> This would probably make a shot of it lethal instead of just enough to get you shitfaced.

I thought that a shot of it was _already_ lethal.

**Comment by ATH1909** [comment]
Still, I'd prefer it if we testmerge this before we fullmerge it, just in case it ruins bartending somehow.

**Comment by granpawalton** [comment]
ahh now ill have to wear a rad suit as engy instead of just chugging screwdrivers

**Comment by Mickyan** [comment]
FYI this has been testmerged on bagil for about a day and the sky hasn't fallen yet

**Pull request merged**